packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -1,5 +1,5 @@
 import { getFieldName, getSubscriptionName, GraphbackOperationType, ModelDefinition, getPrimaryKey, FieldRelationshipMetadata, getDeltaQuery } from '@graphback/core';
-import { GraphbackContext } from '../service/GraphbackContext';
+import { GraphbackContext } from '@graphback/core/src/runtime/GraphbackContext';
```

Comment:
```suggestion
import { GraphbackContext } from '@graphback/core/GraphbackContext';
```

They are exposed here https://github.com/aerogear/graphback/pull/1455/files#diff-621d05b95d70a70d7e4794643ed2d0e9R2